.travis.yml

```diff
@@ -70,6 +72,21 @@ jobs:
     - env: EMBER_TRY_SCENARIO=ember-beta
     - env: EMBER_TRY_SCENARIO=ember-canary
 
+    # runs tests against various open-source projects for early-warning regression analysis
+    - stage: external partner tests
+      env: NAME=model-fragments
+      script: yarn test-external:model-fragments
+    - env: NAME=ember-observer
+      script: yarn test-external:ember-observer
+    - env: NAME=travis-web
+      script: yarn test-external:travis-web
+    - env: NAME=emberaddons.com
+      script: yarn test-external:emberaddons.com
+    - env: NAME=ember-data-change-tracker
+      script: yarn test-external:ember-data-change-tracker
+    - env: NAME=ember-m3
```

Comment:
re all these `NAME=` workarounds, you can now explicitly name jobs using `name: "Whatever"`, as with the “Max Transpilation” one earlier in the file.

Reply:
🗡 🔥 🔋